src/Core/Kernel.php

```diff
@@ -188,9 +188,16 @@ public static function getConnection(): Connection
 
     public function getCacheDir(): string
     {
+        $cacheDir = EnvironmentHelper::getVariable('APP_CACHE_DIR', 'localCache');
```

Comment:
Where does `localCache` come from? Is that a leftover from testing?

Reply:
no it is required to define custom cache directory and is needed for PaaS

Reply:
https://symfony.com/doc/current/configuration/override_dir_structure.html#override-the-cache-directory

Reply:
The default should be getProjectDir as before

Reply:
the intention was to allow relative path as the path on PaaS dedicated is different for every machine

Reply:
e.g.
app/stgl_123afopea/localCache
on another machine it should also work when the path is
app/stgl_456oehgbajs/localCache

Reply:
I assume pr could be closed and adding it in paas-meta recipes with something like:
APP_CACHE_DIR=${PLATFORM_APP_DIR}/localCache
could be more sufficient

Reply:
I will close it as the change in the recipe worked, so not necessary to change it the kernel: https://github.com/shopware/recipes/pull/110/files